.gitignore

```diff
@@ -51,14 +51,6 @@ lib/CMakeFiles/
 lib/cmake_install.cmake
 
 Documentation/Examples/*.generated
-Documentation/Books/Users/book.json
```

Comment:
I suppose book.json is some build artefact? Is it no longer being generated? It's missing in the Books/.gitignore file below